xml/apache_os.xml

```diff
@@ -148,6 +148,13 @@ against the following patterns to extract OS information.
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
+  <fingerprint pattern=".*\(Red Hat Enterprise (?:Linux)?\).*">
+    <description>Red Hat Enterprise Linux</description>
+    <example os.vendor="Red Hat">Apache/2.4.6 (Red Hat Enterprise Linux) OpenSSL/1.0.2k-fips</example>
+    <param pos="0" name="os.vendor" value="Red Hat"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+  </fingerprint>
```

Comment:
To be consistent with other fingerprints the `os.product` should be `Enterprise Linux`, see https://github.com/rapid7/recog/blob/dccf25cfcb92e5b573cc61fd99b8e2089ea410c7/xml/operating_system.xml#L254-L266


Also, please add a CPE entry for this os:
`<param pos="0" name="os.cpe23" value="cpe:/o:redhat:enterprise_linux:-"/>`

This banner may be useful in the `http_servers.xml` file.  If you add it then it would need to be above line 68 (`<description>Apache</description>`) and could likely use much of that fingerprint.